src/docker-osx-dev

```diff
@@ -428,16 +491,18 @@ function get_env_file {
 # Adds environment variables necessary for running Boot2Docker
 #
 function add_environment_variables {
-  local readonly env_file=$(get_env_file)
-  local readonly boot2docker_exports=$(boot2docker shellinit 2>/dev/null)
-  local readonly exports_to_add_to_env_file=$(determine_boot2docker_exports_for_env_file "$boot2docker_exports")
-
-  if [[ ! -z "$exports_to_add_to_env_file" ]]; then
-    log_info "Adding new environment variables to $env_file: $exports_to_add_to_env_file"
-    echo -e "$exports_to_add_to_env_file" >> "$env_file"
-    log_instructions "To pick up important new environment variables in the current shell, run:\n\tsource $env_file"
-  else
-    log_warn "All Boot2Docker environment variables already defined, will not overwrite"
+  if [ -z "$DOCKER_MACHINE_NAME" ]; then
```

Comment:
I haven't used docker-machine much, so just to confirm: it doesn't rely on *any* environment variables?